pkgs/development/python-modules/pyquaternion/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, nose
+}:
+
+buildPythonPackage rec {
+  pname = "pyquaternion";
+  version = "0.9.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-sfYa8hnLL+lmtft5oZISTy5jo/end6w8rfKVexqBvqg=";
+  };
+
+  postPatch = ''
+    echo "${version}" > VERSION.txt
```

Comment:
Why this?

Reply:
It requires this file, it's present on the main repo but not on the pypi tarball

The repo as far as I remember doesn't tag its releases so you have the guesswork to discover what revision the version belongs so using the pypi tarball is preferable

For the nix usecase I think the author did a smart approach xD

Reply:
do you mind adding a comment?

Reply:
https://github.com/KieranWynn/pyquaternion/blob/master/setup.py#L14-L15